docs/setup.md

```diff
@@ -187,6 +187,26 @@ Alternatively you may submit an array of packages to define multiple versions:
 ]
 ```
 Once a custom package has been added to Private Packagist you will benefit from mirroring the zip file. Composer will have an additional location to download the file from if the original storage becomes unavailable.
+
+### Prevent a package from being used with custom packages
+
+While there is no direct "block package" feature in Packagist, you can effectively prevent a specific package from being used by creating a placeholder package. 
```

Comment:
You do mention that on the bottom of this new section, but I felt this was missing up here:
```suggestion
While there is no direct "block package" feature in Packagist, you can effectively prevent a specific package from being used in your project by creating a placeholder package. 
```
What do you think?